### PR TITLE
Support cross building for arm64 Linux

### PR DESCRIPTION
**Pull request by qmfrederik**
This PR adds support for cross-building for arm64 Linux, using the same approach used for arm Linux.

### CODE REVIEW

File: build_projects/dotnet-host-build/CompileTargets.cs [hunk 1]
```diff
@@ -32,9 +32,12 @@ public class CompileTargets
             { "linux-x64", "linux-x64" },
             { "ubuntu.14.04-x64", "ubuntu.14.04-x64" },
             { "ubuntu.16.04-x64", "ubuntu.16.04-x64" },
+            { "ubuntu.16.10-x64", "ubuntu.16.10-x64" },
             { "ubuntu.14.04-arm", "ubuntu.14.04-arm" },
             { "ubuntu.16.04-arm", "ubuntu.16.04-arm" },
-            { "ubuntu.16.10-x64", "ubuntu.16.10-x64" },
+            { "ubuntu.14.04-arm64", "ubuntu.14.04-arm64" },
```
Comment (gkhanna79) [inline build_projects/dotnet-host-build/CompileTargets.cs]: Please comment out these 3 lines - they can only be enabled when we enable official build for Arm64.

File: cross/build-rootfs.sh [hunk 2]
```diff
@@ -16,7 +15,6 @@ __InitialDir=$PWD
 __BuildArch=arm
 __UbuntuArch=armhf
 __UbuntuRepo="http://ports.ubuntu.com/"
-__LLDB_Package="lldb-3.6-dev"
```
Comment (gkhanna79) [inline cross/build-rootfs.sh]: Why do we need to do this?

File: cross/build-rootfs.sh [hunk 3]
```diff
@@ -60,12 +62,6 @@ for i in "$@" ; do
             __UbuntuArch=i386
             __UbuntuRepo="http://archive.ubuntu.com/ubuntu/"
             ;;
-        lldb3.6)
```
Comment (gkhanna79) [inline cross/build-rootfs.sh]: We want to keep the cross-rootfs generation scripts consistent across repos as we use them to generate docker images - please undo this change.

File: cross/build-rootfs.sh (hunk 2 above)
Comment (qmfrederik) [inline cross/build-rootfs.sh]: There's no liblldb-dev for arm64 for Ubuntu 16.10 and earlier (it is present in 17.04, though). I'll revert this change and add the no-lldb option. This makes it consistent with coreclr and corefx.

**Comment by qmfrederik** [comment]
@gkhanna79 Thanks for the feedback, should have been addressed now. Can you take another look?

**Comment by qmfrederik** [comment]
@dotnet test Ubuntu16.10 x64 Release Build please

**Comment by qmfrederik** [comment]
@dotnet-bot test Ubuntu16.10 x64 Release Build please

**Comment by jyoungyun** [comment]
@dotnet-bot test Ubuntu16.10 x64 Release Build  please

File: cross/build-rootfs.sh [hunk 4]
```diff
@@ -2,10 +2,10 @@
 
 usage()
 {
-    echo "Usage: $0 [BuildArch] [LinuxCodeName] [lldbx.y] [--skipunmount]"
```
Comment (jyoungyun) [inline cross/build-rootfs.sh]: We need lldbx.y option :)

File: build_projects/dotnet-host-build/CompileTargets.cs [hunk 5]
```diff
@@ -34,7 +34,8 @@ public class CompileTargets
             { "ubuntu.16.04-x64", "ubuntu.16.04-x64" },
             { "ubuntu.14.04-arm", "ubuntu.14.04-arm" },
             { "ubuntu.16.04-arm", "ubuntu.16.04-arm" },
-            { "ubuntu.16.10-x64", "ubuntu.16.10-x64" },
```
Comment (jyoungyun) [inline build_projects/dotnet-host-build/CompileTargets.cs]: I think this should be restored.

**Comment by hqueue** [comment]
@qmfrederik I think you already updated build-rootfs.sh in corefx. I hope we can keep build-rootfs.sh consistent  across repos.

**Comment by qmfrederik** [comment]
@jyoungyun @hqueue @gkhanna79  Thanks for the feedback, I updated the PR. I think this should be OK to merge now :)

**Comment by jyoungyun** [comment]
LGTM :)

**Comment by qmfrederik** [comment]
@gkhanna79 I think I've addressed all the feedback on this PR, let me know if you need anything else.
